<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="silius" errors="0" failures="65" skipped="13" tests="482" time="249.809" timestamp="2025-03-31T17:14:10.688073+00:00" hostname="fv-az1368-733"><testcase classname="" name="tests.single.gas.test_pre_verification_gas_calculation" time="0.000"><skipped message="collection skipped">('/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/gas/test_pre_verification_gas_calculation.py', 15, 'Skipped: Slow and requires fixing')</skipped></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_priority_fee_bump]" time="0.248" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_max_fee_bump]" time="0.165" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_same_fee]" time="0.191" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_less_fee]" time="0.161" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_below_threshold]" time="0.165" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_at_threshold]" time="0.268" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_above_threshold]" time="0.272" /><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_unstaked_sender" time="0.791" /><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_staked_sender" time="0.932" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_op_access_other_ops_sender_in_bundle" time="0.804" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_sender_double_role_in_bundle" time="0.417" /><testcase classname="tests.single.bundle.test_bundle" name="test_stake_check_in_bundler" time="0.548" /><testcase classname="tests.single.bundle.test_codehash" name="test_codehash_changed" time="0.252"><skipped type="pytest.skip" message="no self destruct. can't check code change..">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_codehash.py:54: no self destruct. can't check code change..</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_paymaster" name="test_paymaster_deposit" time="0.711"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x243710f8b6fdb3f1bcd8ae6b966185a0084da7d0a21c14daa805a728545ffb5e', id=98)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_paymaster.py:48: in test_paymaster_deposit
    assert_rpc_error(res, "too low", RPCErrorCode.PAYMASTER_DEPOSIT_TOO_LOW)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x243710f8b6fdb3f1bcd8ae6b966185a0084da7d0a21c14daa805a728545ffb5e', id=98)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-]" time="0.380" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-CALL:&gt;]" time="0.599" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.694" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-]" time="0.403" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-CALL:&gt;]" time="0.623" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-DELEGATECALL:&gt;]" time="0.721" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-]" time="0.405" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-CALL:&gt;]" time="0.668" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-DELEGATECALL:&gt;]" time="0.718" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-]" time="0.525"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-CALL:&gt;]" time="0.882"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-DELEGATECALL:&gt;]" time="0.865"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-]" time="0.523"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-CALL:&gt;]" time="0.765"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-DELEGATECALL:&gt;]" time="0.860"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-]" time="0.547" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-CALL:&gt;]" time="0.815" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-DELEGATECALL:&gt;]" time="0.897" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-]" time="0.607" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-CALL:&gt;]" time="0.897" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.974" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-]" time="0.434" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-CALL:&gt;]" time="0.666" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.870" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-]" time="0.557" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-CALL:&gt;]" time="0.810" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-DELEGATECALL:&gt;]" time="0.923" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-]" time="0.581" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.785" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.878" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-]" time="0.584" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.867" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.960" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-]" time="1.153" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-CALL:&gt;]" time="1.016" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="1.115" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]unstaked][paymaster][context][ok-]" time="0.344"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='A unstaked paymaster at 0x772e337ace5801a18b1e12b5bba89b7b1bb7171a: must not return context', data=None, id=172)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='A unstaked paymaster at 0x772e337ace5801a18b1e12b5bba89b7b1bb7171a: must not return context', data=None, id=172)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]unstaked][paymaster][context][ok-CALL:&gt;]" time="0.267"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='A unstaked paymaster at 0xb924c65195a22d7a2c6121da7a235a4b846ba3ad: must not return context', data=None, id=174)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='A unstaked paymaster at 0xb924c65195a22d7a2c6121da7a235a4b846ba3ad: must not return context', data=None, id=174)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]unstaked][paymaster][context][ok-DELEGATECALL:&gt;]" time="0.297"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='A unstaked paymaster at 0x7fe507b9df7ed8a7c62a0180c85020f39f9c5922: must not return context', data=None, id=176)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='A unstaked paymaster at 0x7fe507b9df7ed8a7c62a0180c85020f39f9c5922: must not return context', data=None, id=176)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-]" time="0.599" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-CALL:&gt;]" time="0.725" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.822" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-]" time="0.476" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.838" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.784" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-]" time="0.565" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.799" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.783" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-]" time="0.389" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-CALL:&gt;]" time="0.611" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.699" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-]" time="0.513" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-CALL:&gt;]" time="0.740" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-DELEGATECALL:&gt;]" time="0.835" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-]" time="0.520" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-CALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-DELEGATECALL:&gt;]" time="0.851" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-]" time="0.577" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-CALL:&gt;]" time="0.864" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-DELEGATECALL:&gt;]" time="1.029" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.735" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="1.015" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.121" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.743" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="0.921" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.011" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-]" time="0.503" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-CALL:&gt;]" time="0.697" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.797" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-]" time="0.559" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.822" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.917" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-]" time="0.630" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.909" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.994" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-]" time="0.797" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.084" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.159" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]staked][paymaster][context][ok-]" time="0.285" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]staked][paymaster][context][ok-CALL:&gt;]" time="0.411" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050-regression]staked][paymaster][context][ok-DELEGATECALL:&gt;]" time="0.345" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-]" time="0.547" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-CALL:&gt;]" time="0.798" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.892" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-]" time="0.624" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-CALL:&gt;]" time="0.783" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.966" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-]" time="0.565" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.736" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.918" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-]" time="0.505" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.741" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.921" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-]" time="0.323" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-CALL:&gt;]" time="0.498" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.512" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-]" time="0.322" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-CALL:&gt;]" time="0.597" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-DELEGATECALL:&gt;]" time="0.626" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-]" time="0.328" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-CALL:&gt;]" time="0.507" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-DELEGATECALL:&gt;]" time="0.628" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-]" time="0.448" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-CALL:&gt;]" time="0.557" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-DELEGATECALL:&gt;]" time="0.616" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-]" time="0.469" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-CALL:&gt;]" time="0.602" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.665" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-]" time="0.327" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-CALL:&gt;]" time="0.513" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.526" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-]" time="0.334" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-CALL:&gt;]" time="0.550" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-DELEGATECALL:&gt;]" time="0.716" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-]" time="0.349" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-CALL:&gt;]" time="0.599" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.653" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-]" time="0.354" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-CALL:&gt;]" time="0.544" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.686" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-]" time="0.461" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-CALL:&gt;]" time="0.531" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-DELEGATECALL:&gt;]" time="0.537" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-]" time="0.317" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.554" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.622" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-]" time="0.411" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.662" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.721" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-]" time="0.313" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.570" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.628" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-]" time="0.309" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.608" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.663" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-]" time="0.301" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.644" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.590" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-]" time="0.413" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.669" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.725" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-]" time="0.307" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.645" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.608" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-]" time="0.441" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-CALL:&gt;]" time="0.605" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.628" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-]" time="0.366" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.547" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.583" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-]" time="0.352" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-CALL:&gt;]" time="0.521" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.533" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-]" time="0.462" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-CALL:&gt;]" time="0.532" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-DELEGATECALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-]" time="0.361" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-CALL:&gt;]" time="0.540" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-DELEGATECALL:&gt;]" time="0.555" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-]" time="0.382" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-CALL:&gt;]" time="0.635" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-DELEGATECALL:&gt;]" time="0.651" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-]" time="0.486" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-CALL:&gt;]" time="0.733" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.696" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-]" time="0.355" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-CALL:&gt;]" time="0.552" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.556" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-]" time="0.460" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-CALL:&gt;]" time="0.571" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.752" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-]" time="0.372" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-CALL:&gt;]" time="0.625" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.691" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-]" time="0.379" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-CALL:&gt;]" time="0.677" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.742" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-]" time="0.476" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-CALL:&gt;]" time="0.668" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.619" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-]" time="0.566" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-CALL:&gt;]" time="0.732" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.758" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-]" time="0.490" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.575" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.703" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-]" time="0.667" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-CALL:&gt;]" time="0.582" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.472" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-]" time="0.295" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-CALL:&gt;]" time="0.496" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.501" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-]" time="0.305" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-CALL:&gt;]" time="0.638" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.625" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-]" time="0.411" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-CALL:&gt;]" time="0.475" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="0.416" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-]" time="0.858"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-CALL:&gt;]" time="0.783"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
CALL:&gt; funding test_rules_target 0x65D041834D1Caa81FFa3F1fA198a36133E025924

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="0.885"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-]" time="0.907"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.090"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.271"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-]" time="0.340" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.548" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.541" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-]" time="0.419" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-CALL:&gt;]" time="0.529" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.531" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-]" time="0.401" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-CALL:&gt;]" time="0.590" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.592" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-]" time="0.327" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.537" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.632" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-]" time="0.414" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-CALL:&gt;]" time="0.607" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.601" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-]" time="0.423" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-CALL:&gt;]" time="0.534" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.622" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-]" time="0.328" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-CALL:&gt;]" time="0.559" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.551" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-]" time="0.437" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.581" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.616" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-]" time="0.435" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-CALL:&gt;]" time="0.623" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.626" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-]" time="0.354" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.555" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.557" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-]" time="0.347" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-CALL:&gt;]" time="0.574" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.568" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-]" time="0.351" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-CALL:&gt;]" time="0.573" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.657" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-]" time="0.439" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-CALL:&gt;]" time="0.671" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-DELEGATECALL:&gt;]" time="0.701" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-]" time="0.430" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-CALL:&gt;]" time="0.577" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-DELEGATECALL:&gt;]" time="0.555" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-]" time="0.344" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-CALL:&gt;]" time="0.577" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-DELEGATECALL:&gt;]" time="0.574" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-]" time="0.346" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-CALL:&gt;]" time="0.608" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-DELEGATECALL:&gt;]" time="0.715" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-]" time="0.277" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.517" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.560" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-]" time="0.282" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.633" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.522" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-]" time="0.280" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.538" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.526" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-]" time="0.396" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-CALL:&gt;]" time="0.626" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-DELEGATECALL:&gt;]" time="0.517" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-]" time="0.297" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-CALL:&gt;]" time="0.516" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-DELEGATECALL:&gt;]" time="0.641" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-]" time="0.301" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-CALL:&gt;]" time="0.521" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-DELEGATECALL:&gt;]" time="0.620" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-]" time="0.267" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.495" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.494" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-]" time="0.277" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.511" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.514" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-]" time="0.279" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.521" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.518" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-]" time="0.384" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.634" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.578" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-]" time="0.395" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-CALL:&gt;]" time="0.558" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-DELEGATECALL:&gt;]" time="0.557" /><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx" time="0.081"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x84d2…4866 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:36: in test_send_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x84d2…4866 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_initcode" time="0.077"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:85: in test_send_eip_7702_tx_with_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_flag_no_initcode" time="0.081"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:127: in test_send_eip_7702_tx_with_flag_no_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[0]" time="0.080"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x5cb5…8c0a is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x5cb5…8c0a is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[1337]" time="0.079"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x5eac…498d is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x5eac…498d is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_bad_eip_7702_drop_userop" time="0.081"><failure message="AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32602 = Error(code=-32602, message='sender 0x1d1b…e464 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code">tests/single/eip7702/test_eip7702_tuple_userop.py:208: in test_send_bad_eip_7702_drop_userop
    assert_rpc_error(response, "", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32602 = Error(code=-32602, message='sender 0x1d1b…e464 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_nonsender_eip_7702_drop_userop" time="0.163"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x973041d2d14825fd39376af71c7a05a9be76fba6bf9946b976e4c904800f7f4d', id=682)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:223: in test_send_nonsender_eip_7702_drop_userop
    assert_rpc_error(userop.send(), "sender", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x973041d2d14825fd39376af71c7a05a9be76fba6bf9946b976e4c904800f7f4d', id=682)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_wrongchain_eip_7702_drop_userop" time="0.077"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0xbc2e…eba6 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:244: in test_send_wrongchain_eip_7702_drop_userop
    assert_ok(userop.send())
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0xbc2e…eba6 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE]" time="0.342" /><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE2]" time="0.243" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE]" time="0.406" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE2]" time="0.414" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create2_once" time="0.392" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE,expect=ok,rule=OP-032]" time="0.406"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=696)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=696)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE2,expect=err,rule=OP-032]" time="0.296"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f4f133d87b0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f4f133d87b0&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133d8450&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f4f133d8450&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=698).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f4f133d87b0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f4f133d87b0&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133d8450&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f4f133d8450&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=698).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE,expect=ok,rule=EREP-060]" time="0.420"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=700)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=700)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE,expect=err,rule=EREP-060]" time="0.323"><failure message="AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133dacd0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f4f133dacd0&gt; = 'account uses banned opcode: CREATE'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133db9f0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f4f133db9f0&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=702).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133dacd0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f4f133dacd0&gt; = 'account uses banned opcode: CREATE'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133db9f0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f4f133db9f0&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=702).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE2,expect=err,rule=EREP-060]" time="0.320"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f4f133d9a10&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f4f133d9a10&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133d9ad0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f4f133d9ad0&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f4f133d9a10&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f4f133d9a10&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f4f133d9ad0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f4f133d9ad0&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE2,expect=ok,rule=EREP-060]" time="0.419"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-]" time="0.319" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-CALL:&gt;]" time="0.464" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.479" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-]" time="0.312" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-CALL:&gt;]" time="0.369" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.387" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-]" time="0.214" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.476" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.498" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-]" time="0.214" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-CALL:&gt;]" time="0.481" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.501" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-]" time="0.214" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.490" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.405" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-]" time="0.218" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-CALL:&gt;]" time="0.517" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.396" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-]" time="0.325" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.482" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.405" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-]" time="0.328" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-CALL:&gt;]" time="0.489" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.511" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-]" time="0.248" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.412" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.562" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-]" time="0.331" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-CALL:&gt;]" time="0.386" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.401" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-]" time="0.225" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-CALL:&gt;]" time="0.382" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.508" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-]" time="0.332" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.392" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.517" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-]" time="0.343" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-CALL:&gt;]" time="0.394" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.515" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-]" time="0.343" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-CALL:&gt;]" time="0.494" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.521" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-]" time="0.346" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.512" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.435" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-]" time="0.236"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x3999d6b4e30b55d74e0e3ed243f42c5e3052c4573b4bbb31fe9f4dc381713f34', id=798)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x3999d6b4e30b55d74e0e3ed243f42c5e3052c4573b4bbb31fe9f4dc381713f34', id=798)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.499"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x4e7c2825ced84848b0e6d5ae08467601f6964cb5fe8faf099a078fffde77c8cd', id=800)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x4e7c2825ced84848b0e6d5ae08467601f6964cb5fe8faf099a078fffde77c8cd', id=800)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.519"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x35a3995f4b6b7c23d8936f1ee8af438cbb271988b678aea7704ff9207b6a4d7f', id=802)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x35a3995f4b6b7c23d8936f1ee8af438cbb271988b678aea7704ff9207b6a4d7f', id=802)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-]" time="0.337"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x29c04caf56ded998c9a93164c3db46fc059c270ae523a4fa38a0443c14a3e9ff', id=804)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x29c04caf56ded998c9a93164c3db46fc059c270ae523a4fa38a0443c14a3e9ff', id=804)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.538"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x5b9b37ff900427a0949f5e034141a4f00f0e4236fc1d0e22dc3ee7c1df2d869b', id=806)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x5b9b37ff900427a0949f5e034141a4f00f0e4236fc1d0e22dc3ee7c1df2d869b', id=806)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.533"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x36ff154e97199605fe8b8132244b166e0384228ce3e8c4cee8f467c9d813546d', id=808)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x36ff154e97199605fe8b8132244b166e0384228ce3e8c4cee8f467c9d813546d', id=808)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-]" time="0.348"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=810)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=810)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-CALL:&gt;]" time="0.503"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=812)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=812)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-DELEGATECALL:&gt;]" time="0.526"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=814)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=814)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-]" time="0.272"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=816)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=816)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-CALL:&gt;]" time="0.539"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=818)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=818)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-DELEGATECALL:&gt;]" time="0.453"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=820)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=820)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-]" time="0.268" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-CALL:&gt;]" time="0.426" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-DELEGATECALL:&gt;]" time="0.447" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-]" time="0.376" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-CALL:&gt;]" time="0.561" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-DELEGATECALL:&gt;]" time="0.471" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-]" time="0.369" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-CALL:&gt;]" time="0.548" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.632" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-]" time="0.376" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-CALL:&gt;]" time="0.580" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.789" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-]" time="0.296" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.610" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.803" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-]" time="0.295" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-CALL:&gt;]" time="0.605" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.700" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-]" time="0.412" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.635" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.721" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-]" time="0.388" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-CALL:&gt;]" time="0.610" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.584" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-]" time="0.398" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.622" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.711" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-]" time="0.415" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-CALL:&gt;]" time="0.618" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.718" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-]" time="0.456" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.682" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.757" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-]" time="0.398" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-CALL:&gt;]" time="0.644" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.700" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-]" time="0.389" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-CALL:&gt;]" time="0.597" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.686" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-]" time="0.413" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.650" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.749" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-]" time="0.404" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-CALL:&gt;]" time="0.633" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.707" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-]" time="0.419" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-CALL:&gt;]" time="0.624" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.709" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-]" time="0.452" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.687" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.769" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-]" time="0.433"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x420192063f46a7f864187e5ac4d7ff09ee4d7682fe3370826af3ccfe660d42bb', id=924)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x420192063f46a7f864187e5ac4d7ff09ee4d7682fe3370826af3ccfe660d42bb', id=924)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.644"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x4896e9667c6548c5160a285a787173e17ebadd63b5fe48c86b5419dd5e4366de', id=926)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x4896e9667c6548c5160a285a787173e17ebadd63b5fe48c86b5419dd5e4366de', id=926)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.725"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xc420254cb5ddb9c4a519104a4fbb0fadb615fc5d1fd9e83d8a8310770478e8a4', id=928)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xc420254cb5ddb9c4a519104a4fbb0fadb615fc5d1fd9e83d8a8310770478e8a4', id=928)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-]" time="0.482"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x0c06f7eea115faf94ac8a687d7b581761bd6fbacf880742ce7179e0574a8b2f6', id=930)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x0c06f7eea115faf94ac8a687d7b581761bd6fbacf880742ce7179e0574a8b2f6', id=930)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.664"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xc98eea90256be820ae34bd81cb9999b178da81441f1da2cb8a02c5852951e962', id=932)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xc98eea90256be820ae34bd81cb9999b178da81441f1da2cb8a02c5852951e962', id=932)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.747"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xa0f7bb7655b69cbe1fa66fe95751dbdbc6d0e5d9678b253cbcfaecf1f95ad43b', id=934)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xa0f7bb7655b69cbe1fa66fe95751dbdbc6d0e5d9678b253cbcfaecf1f95ad43b', id=934)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[BALANCE]" time="0.528"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=936)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=936)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[SELFBALANCE]" time="0.475"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=938)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=938)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GAS]" time="0.365" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[NUMBER]" time="0.369" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[TIMESTAMP]" time="0.376" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[COINBASE]" time="0.265" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[DIFFICULTY]" time="0.376" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BASEFEE]" time="0.372" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASLIMIT]" time="0.376" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASPRICE]" time="0.375" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFBALANCE]" time="0.401" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BALANCE]" time="0.276" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[ORIGIN]" time="0.281" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOCKHASH]" time="0.389" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE]" time="0.388" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE2]" time="0.392" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFDESTRUCT]" time="0.394" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBHASH]" time="0.394"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x7b52518978339823d65ca9e9593b837d461b83ada36f8353e209025548e5b420', id=970)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x7b52518978339823d65ca9e9593b837d461b83ada36f8353e209025548e5b420', id=970)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBBASEFEE]" time="0.432"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x5ddba84bf36b093eba01577c5db762d447a3374fbc42449f6b47ac3b52dbc8e7', id=972)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x5ddba84bf36b093eba01577c5db762d447a3374fbc42449f6b47ac3b52dbc8e7', id=972)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[BALANCE]" time="0.410"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=974)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=974)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[SELFBALANCE]" time="0.415"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=976)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=976)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_paymaster_on_account_failure" time="0.253"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/reputation/test_erep.py:71: in test_paymaster_on_account_failure
    set_reputation(paymaster.address, ops_seen=5, ops_included=2)
tests/utils.py:321: in set_reputation
    assert res.result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_staked_factory_on_account_failure" time="0.891"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String(&quot;0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000&quot;)))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=986)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/reputation/test_erep.py:119: in test_staked_factory_on_account_failure
    send_bundle_now()
tests/utils.py:230: in send_bundle_now
    assert_ok(RPCRequest(method="debug_bundler_sendBundleNow").send(url))
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String("0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000")))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=986)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[staked]" time="0.485"><failure message="AssertionError: staked account should be blamed instead of paymaster&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 1 diff: Reputation(address='0xcf663a89e5f67163c4b7ee5408086101684d7e5e', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0xcf663a89e5f67163c4b7ee5408086101684d7e5e', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0x562352670efe26d369dc1a3cb550e7b10b88684b',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0xcf663a89e5f67163c4b7ee5408086101684d7e5e',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:174: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: staked account should be blamed instead of paymaster
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 1 diff: Reputation(address='0xcf663a89e5f67163c4b7ee5408086101684d7e5e', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0xcf663a89e5f67163c4b7ee5408086101684d7e5e', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0x562352670efe26d369dc1a3cb550e7b10b88684b',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0xcf663a89e5f67163c4b7ee5408086101684d7e5e',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
staking account &lt;web3._utils.datatypes.Contract object at 0x7f4f1369b2e0&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[unstaked]" time="0.378"><failure message="AssertionError: valid userop. staked paymaster should have opsSeen increment&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 0 diff: Reputation(address='0xbd4834c82aa73cfb79f1eff9bc600fa6d19f5c80', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0xbd4834c82aa73cfb79f1eff9bc600fa6d19f5c80', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0xbd4834c82aa73cfb79f1eff9bc600fa6d19f5c80',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0xd38ee6d984d140ad9b7ca5ed52e839716a99b979',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:156: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: valid userop. staked paymaster should have opsSeen increment
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 0 diff: Reputation(address='0xbd4834c82aa73cfb79f1eff9bc600fa6d19f5c80', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0xbd4834c82aa73cfb79f1eff9bc600fa6d19f5c80', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0xbd4834c82aa73cfb79f1eff9bc600fa6d19f5c80',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0xd38ee6d984d140ad9b7ca5ed52e839716a99b979',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
unstaked account &lt;web3._utils.datatypes.Contract object at 0x7f4f13b16020&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_replace" name="test_replace_paymaster" time="0.585"><failure message="AssertionError: replace with unmodified userop&#10;assert 2 == 1&#10; +  where 2 = pm_opsSeen('0x3e904b12Cc3e6F3Df436a4C1825d2E771B3ad36D')&#10; +    where '0x3e904b12Cc3e6F3Df436a4C1825d2E771B3ad36D' = &lt;web3._utils.datatypes.Contract object at 0x7f4f13520280&gt;.address">tests/single/reputation/test_replace.py:37: in test_replace_paymaster
    assert pm_opsSeen(pm1.address) == 1, "replace with unmodified userop"
E   AssertionError: replace with unmodified userop
E   assert 2 == 1
E    +  where 2 = pm_opsSeen('0x3e904b12Cc3e6F3Df436a4C1825d2E771B3ad36D')
E    +    where '0x3e904b12Cc3e6F3Df436a4C1825d2E771B3ad36D' = &lt;web3._utils.datatypes.Contract object at 0x7f4f13520280&gt;.address</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[with_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[without_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_chainId" name="test_eth_chainId[]" time="0.009" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas[]" time="0.177" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_execution_revert" time="0.177"><failure message="AttributeError: expected error object, got:&#10;Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1014)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/rpc/test_eth_estimateUserOperationGas.py:32: in test_eth_estimateUserOperationGas_execution_revert
    assert_rpc_error(response, "", RPCErrorCode.EXECUTION_REVERTED)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1014)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_simulation_revert" time="0.029"><failure message="assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: &quot;AA23 reverted&quot;,...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1016).code">tests/single/rpc/test_eth_estimateUserOperationGas.py:40: in test_eth_estimateUserOperationGas_simulation_revert
    assert_rpc_error(response, "dead signature", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: "AA23 reverted",...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1016).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash[]" time="0.360"><failure message="jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas&#10;&#10;Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:&#10;    {'title': 'paymaster verification gas limit',&#10;     'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},&#10;               {'type': 'number'}],&#10;     'description': 'The amount of gas to allocate for the paymaster '&#10;                    'validation code'}&#10;&#10;On instance['userOperation']['paymasterVerificationGasLimit']:&#10;    None">tests/single/rpc/test_eth_getUserOperationByHash.py:24: in test_eth_getUserOperationByHash
    validate(instance=response.result, schema=schema)
.venv/lib/python3.10/site-packages/jsonschema/validators.py:1332: in validate
    raise error
E   jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas
E   
E   Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:
E       {'title': 'paymaster verification gas limit',
E        'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},
E                  {'type': 'number'}],
E        'description': 'The amount of gas to allocate for the paymaster '
E                       'validation code'}
E   
E   On instance['userOperation']['paymasterVerificationGasLimit']:
E       None</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash_error" time="0.005" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt[]" time="0.374" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt_error" time="0.005" /><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation[]" time="0.338"><failure message="AssertionError: assert '0x620709574c...90c24051f4bdb' == '0x08dfdd04f8...e96fe7cc35041'&#10;  &#10;  - 0x08dfdd04f8d055467d131641828b662c9ffec94167273057150e96fe7cc35041&#10;  + 0x620709574ca83c21c5377ca2cf1e66df8fe6e67b301d2d8eac590c24051f4bdb">tests/single/rpc/test_eth_sendUserOperation.py:19: in test_eth_sendUserOperation
    assert response.result == userop_hash(helper_contract, userop)
E   AssertionError: assert '0x620709574c...90c24051f4bdb' == '0x08dfdd04f8...e96fe7cc35041'
E     
E     - 0x08dfdd04f8d055467d131641828b662c9ffec94167273057150e96fe7cc35041
E     + 0x620709574ca83c21c5377ca2cf1e66df8fe6e67b301d2d8eac590c24051f4bdb</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_revert" time="0.043"><failure message="AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1033).code">tests/single/rpc/test_eth_sendUserOperation.py:32: in test_eth_sendUserOperation_revert
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1033).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_invalid_signature" time="0.032"><failure message="AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;&#10; +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1036).code">tests/single/rpc/test_eth_sendUserOperation.py:39: in test_eth_sendUserOperation_invalid_signature
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;
E    +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1036).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_supportedEntryPoints" name="test_eth_supportedEntryPoints[]" time="0.010" /></testsuite></testsuites>